README.md

```diff
@@ -620,3 +626,7 @@ to the string value `production`.
 Copyright (c) 2016 Khan Academy
 
 [webcomponents]: http://w3c.github.io/webcomponents/spec/custom
+
+[gzip-badge]: http://img.badgesize.io/https://unpkg.com/aphrodite/dist/aphrodite.umd.min.js?compression=gzip&label=gzip%20size
+[size-badge]: http://img.badgesize.io/https://unpkg.com/aphrodite/dist/aphrodite.umd.min.js?label=size
```

Comment:
I'm not entirely sure this will be representative of the bundle size impact for most people. For instance, it looks like this file shows ~24 KiB, but in practice aphrodite is closer to 13 KiB in my bundle.

![screen shot 2017-10-12 at 8 25 47 am](https://user-images.githubusercontent.com/195534/31504351-05cba524-af27-11e7-8a26-41ebbd860770.png)

Although now that I think about it some more, this probably also includes all dependencies, such as inline-style-prefixer, which might all add up to somewhere near this size in the end.